charts/sessionspaces/templates/clusterrole.yaml

```diff
@@ -0,0 +1,39 @@
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  name: {{ include "sessionspaces.fullname" . }}
+rules:
+- apiGroups: [""]
+  resources: ["pods", "services"]
+  verbs: ["get", "list", "watch", "patch", "create"] 
+- apiGroups: [""]
+  resources: ["namespaces"]
+  verbs: ["create", "delete", "get", "list"]
+- apiGroups: [""]
+  resources: ["pods/exec"]
+  verbs: ["create"]
+- apiGroups: [""]
+  resources: ["pods/logs"]
+  verbs: ["get", "watch"]
```

Comment:
Does sessionspaces need permission to exec into pods (`pods/exec`) or read pods logs (`pods/logs`)?

Reply:
It creates a argo-workflows ClusterRole with those resources